### PR TITLE
Feature/pantry class testing

### PR DESCRIPTION
**Pull request by bhohnco**
# Pull Request Template
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant screenshots of your code being deployed. 
Summary - created pantry class and methods, and also created testing file. Troubleshooted issues with the team, and was able to fix test errors. All methods are now passing the tests. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)
## Checklist:
- [X ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ X] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [X ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: test/pantry-testing-data.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+const pantryTestingData = [
```
Comment (RMartin0717) [inline test/pantry-testing-data.js:1]: Awesome job putting the test data in its own file

Comment (michannstoner) [reply test/pantry-testing-data.js:1]: Agreed with Riley here, appreciate the test data file for a much cleaner test file.

File: src/pantry.js [hunk 2]
```diff
@@ -1,7 +1,67 @@
...
+          item.amount -= ingredient.quantity.amount
+        }
+      }))
+    });
+    this.contents = this.contents.filter((item) => {
+      return item.amount > 0;
+    })
+  }
+
+  checkPantryForRequiredIngredients(recipe) {
```
Comment (michannstoner) [inline src/pantry.js:41]: Nice work with this method. I like the reduce nested inside of the forEach!

**Review by michannstoner — APPROVED** [review]
Looks great, thank you for your hard work Bryan!

**Pull request merged**